mailbox/store/src/main/java/org/apache/james/mailbox/store/mail/model/impl/SimpleMailbox.java

```diff
@@ -160,4 +161,10 @@ public void setACL(MailboxACL acl) {
         this.acl = acl;
     }
 
+    @Override
+    public boolean isChildOf(Mailbox potentialParent, MailboxSession mailboxSession) {
+        return namespace.equals(potentialParent.getNamespace())
+            && user.equals(potentialParent.getUser())
+            && name.startsWith(potentialParent.getName() + mailboxSession.getPathDelimiter());
```

Comment:
I don't see my comment... Is'nt it possible to not duplicate this code several times?